index.android.js

```diff
@@ -420,8 +420,8 @@ class RNGoogleFit {
             if (options.unit === 'pound') {
               el.value = KgToLbs(el.value) //convert back to pounds
             }
-            el.startDate = new Date(el.startDate).toISOString()
-            el.endDate = new Date(el.endDate).toISOString()
+            el.startDate = moment.parseZone(el.startDate).toISOString(true)
+            el.endDate = moment.parseZone(el.endDate).toISOString(true)
```

Comment:
The reason I used `moment` here is because that's what is used within the `prepareResponse` function which most of our health sync values use. I don't think we want to deviate to far away from the main library in this case.